### PR TITLE
Fix missing signature files when publishing to the plugin portal

### PR DESCRIPTION
**Pull request by alextu**

### CODE REVIEW

**Review by etiennestuder — APPROVED** [review]
LGTM. Some of the GH action builds are failing, not sure if it is related to your change or not.

**Comment by alextu** [comment]
> LGTM. Some of the GH action builds are failing, not sure if it is related to your change or not.

Unfortunately it looks related, will investigate

**Comment by runningcode** [comment]
A very quick glance seems to indicate that the publication to the mavenLocal repository which is used by the test isn't working right.

**Comment by etiennestuder** [comment]
I will bail out of the review here. @ghale and @runningcode are better suited to verify this.

**Comment by alextu** [comment]
> A very quick glance seems to indicate that the publication to the mavenLocal repository which is used by the test isn't working right.

Yes, I've just made a change that should fix it. Will wait for the "Verify Build" to confirm.

File: build.gradle [hunk 1]
```diff
@@ -80,6 +82,15 @@ pluginBundle {
 // in the TestKit plugin classpath.
 def localRepo = file("$buildDir/local-repo")
 publishing {
+    publications {
+        maven(MavenPublication) {
```
Comment (runningcode) [inline build.gradle:86]: Can you add a comment here if we can remove this section in the future when the plugin portal automatically supports publishing the signature files? That way we aren't wondering if this should be removed.

**Review by runningcode — APPROVED** [review]

**Comment by runningcode** [comment]
Looks good, thanks!

File: build.gradle (hunk 1 above)
Comment (alextu) [reply build.gradle:89]: done

**Pull request merged**